api/envoy/service/discovery/v3/discovery.proto

```diff
@@ -106,6 +107,16 @@ message DiscoveryResponse {
   // [#not-implemented-hide:]
   // The control plane instance that sent the response.
   config.core.v3.ControlPlane control_plane = 6;
+
+  // Time-to-live value for all resources returned in the response. For each resource, a timer is started. The timer
```

Comment:
"For each resource, a timer is started." from my reading this implies per-resource timers when I think we agreed we are doing a TTL for all resources? Can you potentially clarify?